sig-scalability/charter.md

```diff
@@ -0,0 +1,192 @@
+# SIG Scalability Charter
+
+## Mission
+The SIG Scalability helps to define scalability goals for Kubernetes, ensures
+that they all play well together and ensures that every Kubernetes release meets
+them by measuring performance and scalability indicators and publishing the
+results.
+
+We also coordinate and contribute to general system-wide scalability and
+performance improvements (that don’t fall into the charter of another individual
+SIG) as well as provide consultations about any scalability and performance
+related aspects of Kubernetes.
+
+## What can we do/require from other SIGs
+Scalability and performance are horizontal aspects of the system - changes in a
+single place of Kubernetes may affect the whole system. As a result, to
+effectively ensure Kubernetes scales, we need a special cross-SIG privileges.
+
+- We can rollback any merged PR if it has been identified as a cause of an SLO
```

Comment:
can you link to the SLOs? Where are they? Who defines them? Is the API machinery one below the only example?